tools/genssl

```diff
@@ -120,6 +120,7 @@ __GNUTLS_END__
 	$status ||= system "$certtool --generate-privkey $sec_param --outfile key.pem";
 	$status ||= system "$certtool --generate-self-signed --load-privkey key.pem --outfile cert.pem --template $tmp";
 	$status ||= system "$certtool --generate-dh-params $sec_param --outfile dhparams.pem";
+	$status ||= system "$certtool --generate-request $sec_param --load-privkey key.pem --outfile csr.pem --template $tmp";
```

Comment:
I'm not sure whether $sec_param is required here. What do you think @attilamolnar?